tools/invoke-tests

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+
+# process arguments
+if [[ $1 == --coverage ]]; then
```

Comment:
As you already decided for a full-blown bash script I suggest to use getopt instead of the "manual" argument parsing, e.g. see https://github.com/os-autoinst/openQA/blob/master/script/configure-web-proxy#L14 for an example to follow. This can also help with the command line as used in Makefile.am as I consider `invoke-tests prove make $(abs_builddir)` not very clear as well

Reply:
Yes, I should make the argument parsing a bit more sophisticated.